ui/forms.py

```diff
@@ -52,24 +52,38 @@ def to_python(self, values):
         return [self.to_model(value) for value in values]
 
 
+class CollectionForm(ModelForm):
+    """
+    Form to create a collection
+    """
+    title = forms.CharField()
+    owner = forms.CharField(widget=forms.HiddenInput)
+
+    class Meta:
+        model = models.Collection
+        fields = ('title', 'description', 'owner', )
+
+
 class VideoForm(ModelForm):
     """
     Video Form
     """
     moira_lists = DynamicModelMultipleChoiceField(
-        model=MoiraList, field="name",
+        model=models.MoiraList,
```

Comment:
Maybe `moira_lists` should be associated with collections instead of videos now?